### PR TITLE
3206 eventcard limit description lengths

### LINKED ISSUE #3206: [EventCard] limit description lengths
### Feature description

Currently, we allow cards to have a description by rendering content given in the `<slot>` outlet. 

However, sometimes the content can become really long; this not only throws off the layout from other cards, but it creates inconsistency.

We should figure out a way to limit the description passed into the event cards, either by limiting the content in the data, or figuring out some method of truncation.

### Screenshot

Screenshot 2023-05-11 at 12 53 50 PM (2)

### Card content guidelines
[Figma board](https://www.figma.com/file/025EpGKBt27LQXuoFOhCp2/Qiskit-Card-content-guidelines?type=design&node-id=222-1591&mode=design&t=cLbspavCOv7fd3Mq-4)

### Additional context

This issue might occur on other card implementations as well, but it's most evident in event cards.

### PR DESCRIPTION
**Pull request by paaragon**
## Changes

Closes #3206.

Descriptions in EventsCard have now a maximum length. If this max-length is exceeded, it is truncated and a "show more" is displayed.

If the "show more" is clicked, the entire length and a "show less" is displayed.

## How to read this PR

Go to `/events` and test the "show more"/"show less" functionality on the EventsCards

## Screenshots

- Show more:
image

- Show less:
image

### CODE REVIEW

**Review by techtolentino — APPROVED** [review]
Thank you for working on this @paaragon 


Non-blocking comment:
my only question is if we need this [margin-bottom here](https://github.com/Qiskit/qiskit.org/pull/3484/files#diff-7faf88c049368c00f5d24443db76a9fc8d769ce23f1274466a7fdfe84557587eR266)

I like the overall card layout spacing better without it:


https://github.com/Qiskit/qiskit.org/assets/6276074/66eef68f-e0bf-4c49-a6c0-6a23f5f9feeb

**Comment by eddybrando** [comment]
> Thank you for working on this @paaragon
> 
> Non-blocking comment: my only question is if we need this [margin-bottom here](https://github.com/Qiskit/qiskit.org/pull/3484/files#diff-7faf88c049368c00f5d24443db76a9fc8d769ce23f1274466a7fdfe84557587eR266)
> 
> I like the overall card layout spacing better without it:
> 
>  Screen.Recording.2023-09-07.at.10.16.02.AM.mov

Good catch @techtolentino! Now we had too much spacing.

I'm removing that extra spacing now.

File: components/Events/EventsCard.vue [hunk 1]
```diff
@@ -68,18 +67,17 @@ withDefaults(defineProps<Props>(), {
...
   verticalLayout: false,
+  description: undefined,
+  to: "",
   regions: () => [],
 });
 </script>
 
 <style lang="scss" scoped>
 @use "~/assets/scss/carbon.scss";
+@use "~/assets/scss/helpers/index.scss" as qiskit;
```
Comment (eddybrando) [inline components/Events/EventsCard.vue:78]: This is not used.

```suggestion
```

**Review by eddybrando — DISMISSED** [review]
Thank you very much @paaragon!

**Review by eddybrando — APPROVED** [review]

**Pull request merged**